cmd/create/execution.go

```diff
@@ -72,6 +72,12 @@ The root project and domain flags of -p and -d should point to task/launch plans
 
  flytectl create execution --execFile execution_spec.yaml -p flytectldemo -d development --targetProject flytesnacks
 
+Also an execution can be relaunched by passing in current execution id.
+
+::
+
+ flytectl create execution --relaunch ffb31066a0f8b4d52b77 -p flytectldemo -d development
```

Comment:
can you also supply a new name as the argument - Guess you can right as you do in regular launch execution?

Reply:
Will be doing in a followup PR.